src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/LotServiceImpl.java

```diff
@@ -111,7 +112,7 @@ public long countSearchLots(final LotsSearchDto lotsSearchDto) {
 	}
 
 	@Override
-	public List<UserDefinedField> getGermplasmAttributeTypes(final LotsSearchDto searchDto) {
+	public List<CVTerm> getGermplasmAttributeTypes(final LotsSearchDto searchDto) {
```

Comment:
@cuenyad 
Do not expose entities in BMSAPI, please use corresponding DTO

Reply:
done.